### ISSUE TITLE
[Bug]: Unable to load world with Chipped 3.0.0 in Minecraft 1.20.1 [FIXED]

### ISSUE THREAD

**Issue by meadowsjared** [CookieCNetwork/Chipped#216]
### Bug Description

Loading the bare minimum I need to run Chipped, which are these mods:
```
athena-fabric-1.20.1-3.0.0
chipped-fabric-1.20.1-3.0.0
fabric-api-0.84.0+1.20.1
resourcefullib-fabric-1.20.1-2.1.1
```
I'm getting 
`Bandayd lost connection: Invalid player data`
Note: I got chipped-fabric from:
https://modrinth.com/mod/chipped/version/3.0.0

See log below for details...

### How to Reproduce?

1. Install:
`athena-fabric-1.20.1-3.0.0`
https://modrinth.com/mod/athena-ctm/version/3.0.0
`chipped-fabric-1.20.1-3.0.0`
`fabric-api-0.84.0+1.20.1`
`resourcefullib-fabric-1.20.1-2.1.1`
https://modrinth.com/mod/resourceful-lib/version/5gAcCZfI
2. create a new world
3. try and load the new world
4. you will see error:
`{YourName} lost connection: Invalid player data`

### Expected Behavior

The error should not happen, and the world should load.

### Version

3.0.0

### Mod Loader Version

fabric-loader-0.14.21-1.20.1

### Logs or additional context

```log
[23:58:01] [main/INFO]: Loading Minecraft 1.20.1 with Fabric Loader 0.14.21
[23:58:01] [main/INFO]: Loading 57 mods:
	- athena 3.0.0
	- chipped 3.0.0
	- fabric-api 0.84.0+1.20.1
	   |-- fabric-api-base 0.4.29+b04edc7a77
	   |-- fabric-api-lookup-api-v1 1.6.34+4d8536c977
	   |-- fabric-biome-api-v1 13.0.10+b3afc78b77
	   |-- fabric-block-api-v1 1.0.9+e022e5d177
	   |-- fabric-blockrenderlayer-v1 1.1.39+b3afc78b77
	   |-- fabric-client-tags-api-v1 1.0.20+b3afc78b77
	   |-- fabric-command-api-v1 1.2.32+f71b366f77
	   |-- fabric-command-api-v2 2.2.11+b3afc78b77
	   |-- fabric-commands-v0 0.2.49+df3654b377
	   |-- fabric-containers-v0 0.1.61+df3654b377
	   |-- fabric-content-registries-v0 4.0.7+b3afc78b77
	   |-- fabric-convention-tags-v1 1.5.3+b3afc78b77
	   |-- fabric-crash-report-info-v1 0.2.18+aeb40ebe77
	   |-- fabric-data-generation-api-v1 12.1.11+b3afc78b77
	   |-- fabric-dimensions-v1 2.1.51+b3afc78b77
	   |-- fabric-entity-events-v1 1.5.21+b3afc78b77
	   |-- fabric-events-interaction-v0 0.6.0+b3afc78b77
	   |-- fabric-events-lifecycle-v0 0.2.61+df3654b377
	   |-- fabric-game-rule-api-v1 1.0.38+b04edc7a77
	   |-- fabric-item-api-v1 2.1.26+b3afc78b77
	   |-- fabric-item-group-api-v1 4.0.7+b3afc78b77
	   |-- fabric-key-binding-api-v1 1.0.36+fb8d95da77
	   |-- fabric-keybindings-v0 0.2.34+df3654b377
	   |-- fabric-lifecycle-events-v1 2.2.20+b3afc78b77
	   |-- fabric-loot-api-v2 1.1.37+b3afc78b77
	   |-- fabric-loot-tables-v1 1.1.41+9e7660c677
	   |-- fabric-message-api-v1 5.1.6+b3afc78b77
	   |-- fabric-mining-level-api-v1 2.1.47+b3afc78b77
	   |-- fabric-models-v0 0.3.35+b3afc78b77
	   |-- fabric-networking-api-v1 1.3.8+b3afc78b77
	   |-- fabric-networking-v0 0.3.48+df3654b377
	   |-- fabric-object-builder-api-v1 11.0.6+b3afc78b77
	   |-- fabric-particles-v1 1.0.28+b3afc78b77
	   |-- fabric-recipe-api-v1 1.0.18+b3afc78b77
	   |-- fabric-registry-sync-v0 2.2.6+b3afc78b77
	   |-- fabric-renderer-api-v1 3.1.0+c154966e77
	   |-- fabric-renderer-indigo 1.4.0+c154966e77
	   |-- fabric-renderer-registries-v1 3.2.44+df3654b377
	   |-- fabric-rendering-data-attachment-v1 0.3.33+b3afc78b77
	   |-- fabric-rendering-fluids-v1 3.0.26+b3afc78b77
	   |-- fabric-rendering-v0 1.1.47+df3654b377
	   |-- fabric-rendering-v1 3.0.6+b3afc78b77
	   |-- fabric-resource-conditions-api-v1 2.3.5+ea08f9d877
	   |-- fabric-resource-loader-v0 0.11.7+f7923f6d77
	   |-- fabric-screen-api-v1 2.0.6+b3afc78b77
	   |-- fabric-screen-handler-api-v1 1.3.27+b3afc78b77
	   |-- fabric-sound-api-v1 1.0.12+b3afc78b77
	   |-- fabric-transfer-api-v1 3.2.2+b3afc78b77
	   \-- fabric-transitive-access-wideners-v1 4.2.0+b3afc78b77
	- fabricloader 0.14.21
	- java 17
	- minecraft 1.20.1
	- resourcefullib 2.1.1
	   \-- com_teamresourceful_yabn 1.0.3
[23:58:16] [Render thread/INFO]: Setting user: Bandayd
lots of these...
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2248 doesn't.
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2465 doesn't.
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2248 doesn't.
[23:58:16] [Render thread/ERROR]: Item classes should end with Item and class_1747 doesn't.
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2248 doesn't.
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2465 doesn't.
[23:58:16] [Render thread/ERROR]: Block classes should end with Block and class_2248 doesn't.
[23:58:18] [Render thread/ERROR]: Block classes should end with Block and class_2465 doesn't.
[23:58:18] [Render thread/ERROR]: Block classes should end with Block and class_2248 doesn't.
[23:58:18] [Render thread/ERROR]: Item classes should end with Item and class_1747 doesn't.
[23:58:18] [Render thread/ERROR]: Item classes should end with Item and class_1747 doesn't.
[23:58:18] [Render thread/ERROR]: Block classes should end with Block and class_3749 doesn't.
[23:58:18] [Render thread/ERROR]: Item classes should end with Item and class_1747 doesn't.
[23:58:18] [Render thread/ERROR]: Block classes should end with Block and class_3749 doesn't.
[23:58:18] [Render thread/ERROR]: Block classes should end with Block and class_3749 doesn't.
[23:58:18] [Render thread/ERROR]: Item classes should end with Item and class_1747 doesn't.
[23:59:49] [progressListener/INFO]: Preparing spawn area: 96%
[23:59:49] [progressListener/INFO]: Time elapsed: 23991 ms
[23:59:50] [Server thread/INFO]: Changing view distance to 12, from 10
[23:59:50] [Server thread/INFO]: Changing simulation distance to 12, from 0
[23:59:51] [Server thread/INFO]: Bandayd[local:E:8d9198a6] logged in with entity id 157 at (-1.5, 68.0, 4.5)
[23:59:51] [Server thread/ERROR]: Couldn't place player in world
java.lang.IllegalArgumentException: Unrecognized argument type net.minecraft.class_4530@590a7f10 (class net.minecraft.class_4530)
	at net.minecraft.class_2316.method_41983(class_2316.java:145) ~[client-intermediary.jar:?]
	at net.minecraft.class_2316.method_41985(class_2316.java:151) ~[client-intermediary.jar:?]
	at net.minecraft.class_2641$class_7232.<init>(class_2641.java:162) ~[client-intermediary.jar:?]
	at net.minecraft.class_2641.method_11401(class_2641.java:235) ~[client-intermediary.jar:?]
	at net.minecraft.class_2641.method_30945(class_2641.java:107) ~[client-intermediary.jar:?]
	at net.minecraft.class_2641.<init>(class_2641.java:52) ~[client-intermediary.jar:?]
	at net.minecraft.class_2170.method_9241(class_2170.java:341) ~[client-intermediary.jar:?]
	at net.minecraft.class_3324.method_14596(class_3324.java:665) ~[client-intermediary.jar:?]
	at net.minecraft.class_3324.method_14576(class_3324.java:566) ~[client-intermediary.jar:?]
	at net.minecraft.class_3324.method_14570(class_3324.java:221) ~[client-intermediary.jar:?]
	at net.minecraft.class_3248.method_33800(class_3248.java:131) ~[client-intermediary.jar:?]
	at net.minecraft.class_3248.method_14384(class_3248.java:118) ~[client-intermediary.jar:?]
	at net.minecraft.class_3248.redirect$zfa000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563) ~[client-intermediary.jar:?]
	at net.minecraft.class_3248.method_18784(class_3248.java:68) ~[client-intermediary.jar:?]
	at net.minecraft.class_2535.method_10754(class_2535.java:259) ~[client-intermediary.jar:?]
	at net.minecraft.class_3242.method_14357(class_3242.java:172) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:908) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:105) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[23:59:51] [Server thread/INFO]: Bandayd lost connection: Invalid player data
[23:59:51] [Server thread/INFO]: Bandayd left the game
[23:59:51] [Server thread/INFO]: Stopping singleplayer server as player logged out
[23:59:51] [Server thread/INFO]: Stopping server
[23:59:51] [Server thread/INFO]: Saving players
[23:59:51] [Server thread/INFO]: Saving worlds
[23:59:53] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Chipped]'/minecraft:overworld
[23:59:54] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Chipped]'/minecraft:the_end
[23:59:54] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Chipped]'/minecraft:the_nether
[23:59:54] [Server thread/INFO]: ThreadedAnvilChunkStorage (Chipped): All chunks are saved
[23:59:54] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[23:59:54] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[23:59:54] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
```

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by ThatGravyBoat** [comment]
WE are pushing a fix now

**Comment by AlexNijjar** [comment]
yep

**Comment by meadowsjared** [comment]
Thank you! ❤️ 

Also, I can confirm this is fixed in 3.0.1

Thank you so much for
1. updating it
2. supporting it
3. fixing this so quickly!
You rock! ❤️
